chain/client/src/client.rs

```diff
@@ -187,6 +188,8 @@ pub struct Client {
     /// Tracks current chunks that are ready to be included in block
     /// Also tracks banned chunk producers and filters out chunks produced by them
     pub chunk_inclusion_tracker: ChunkInclusionTracker,
+    /// Tracks chunk endorsements received from chunk validators. Used to filter out chunks ready for inclusion
+    pub chunk_endorsement_tracker: EndorsementTracker,
```

Comment:
Moved LRU cache tracking chunk endorsements to a separate struct. No functional changes